man/systemd.service.xml

```diff
@@ -255,6 +255,31 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ExitType=</varname></term>
+
+        <listitem>
+          <para>Configures the process exit type for this service unit. One of <option>main</option> or
+          <option>cgroup</option>:</para>
+
+          <itemizedlist>
+            <listitem><para>If set to <option>main</option> (the default), the service manager
+            will consider the unit stopped when the main process, which is determined according to the
+            <varname>Type=</varname>, exits. Consequently, it cannot be used with
+            <varname>Type=</varname><option>oneshot</option>.</para></listitem>
+
+            <listitem><para>The <option>cgroup</option> exit type is meant for applications whose forking model is not
+            known ahead of time and which might not have a specific main process. The service will stay running as long
```

Comment:
Please drop the explanatory part of the first sentence here. Reference first, explanations laters.
Also maybe "stay running" → "be considered running".

`When  <option>cgroup</option>, the service will be considered running as long as at least one process in the cgroup has not exited.` ?

Reply:
Right. I'll merge the explanation with the rest below then.